workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -2201,6 +2355,18 @@ def main():
     scale_down_timeout = int(api.retrieve_preference('ge.autoscaling.scale.down.timeout', default_value=30))
     scale_up_polling_timeout = int(api.retrieve_preference('ge.autoscaling.scale.up.polling.timeout',
                                                            default_value=900))
+    common_utils = ScaleCommonUtils(api=api)
+    static_host_storage = FileSystemHostStorage(cmd_executor=cmd_executor,
+                                                storage_file=os.path.join(work_dir,
+                                                                          '.static.%s.storage' % queue_name),
+                                                clock=clock)
+    run_activation_timeout = 30
+    if static_hosts_number:
+        init_static_hosts(api=api, parent_run_id=master_run_id, common_utils=common_utils,
+                          static_host_storage=static_host_storage, clock=clock,
+                          run_activation_timeout=run_activation_timeout)
+    if not autoscale_enabled:
+        autoscaling_hosts_number = 0
```

Comment:
Let's add a small log message just to be clear that this is a special autoscaler launch.

```suggestion
    if not autoscale_enabled:
        Logger.info('Using non autoscaling mode...')
        autoscaling_hosts_number = 0
```